ci-build.sh

```diff
@@ -43,9 +43,9 @@ case ${BUILD_TYPE} in
     ./gradlew \
       -Dorg.gradle.jvmargs="${JVM_ARGUMENTS}" \
       -Dorg.gradle.daemon=false \
```

Comment:
Minor quibble but I think you should remove the `-Dorg.gradle.daemon=false` line. I'm not sure this is needed or if it's even doing anything.